Documentation/ApiOverview/BackendModules/TemplateClass/Index.rst

```diff
@@ -43,7 +43,7 @@ controller:
        protected ModuleTemplateFactory $moduleTemplateFactory;
 
        public function __construct(
-           ModuleTemplateFactory $moduleTemplateFactory,
+           ModuleTemplateFactory $moduleTemplateFactory
```

Comment:
Just FYI: technically this allowed in PHP 8 but TYPO3v11 also supports PHP 7.4 so we should stick to that.